LibraBFT/Impl/Consensus/ConsensusTypes/ExecutedBlock.agda

```diff
@@ -12,6 +14,15 @@ open import Optics.All
 
 module LibraBFT.Impl.Consensus.ConsensusTypes.ExecutedBlock where
 
+blockInfo : ExecutedBlock → BlockInfo
```

Comment:
Here again you will encounter minor conflicts if I get #74 in ahead of you.

Reply:
No problem.